docs/software-packages/lammps.md

```diff
@@ -21,6 +21,12 @@ users. A number of versions are available:
 - Parallel MPI: using intel-20.4 mpi
 - MPI+GPU version: using intel-20.4 mpi and CUDA11.8
 
+To see what versions are currently available:
+```
+  $ module avail lammps
+```
+The current default central module is based on the ``08Feb2023`` release.
```

Comment:
It's version 15Dec2023 -- or update 2 to 02Aug2023

Reply:
done.